### DISCUSSION TITLE
Doordash interview

### DISCUSSION THREAD

**Discussion by tobechinw** [jenndryden/Canadian-Tech-Internships-and-New-Grad-2025#162] [General]
Has anyone had an interview with Doordash this recruiting season? If so, have you heard back yet ?

**Comment by Khalid0md** [comment]
I did, got rejected, didn't do great in the interview tbh

**Reply by tobechinw** [reply]
Sorry man, how long did it take to hear back?

**Reply by Khalid0md** [reply]
about a week, did it oct 6 too I think

**Comment by jam-sha** [comment]
I interviewed on Oct 6 and didn't hear back yet, when did you interview?

**Reply by tobechinw** [reply]
Oh wow okay, I interviewed on Friday last week.

**Reply by tobechinw** [reply]
How’d you do on the interview?

**Reply by jam-sha** [reply]
I solved both brute force (got lc hard questions for both... my luck) and got partly through optimal, did really good on behavioural. Fyi I know many people who have interviewed in the past 2 weeks and none have heard back yet, I assume they are just busy.

**Reply by tobechinw** [reply]
damn 2 hards is rough, I hope we get offers though!

**Reply by jam-sha** [reply]
@tobechinw I got rejected yesterday. Good luck man.

**Reply by tobechinw** [reply]
same, it hurts man i really had so much hope

**Comment by ShevinuM** [comment]
You guys get interviews 😭

**Reply by Khalid0md** [reply]
you gotta apply like there's no tomorrow, 10 in 50 will give you OA, 5 will give interview, 1 might offer

**Reply by ShevinuM** [reply]
I don't see a lot of postings in Canada either on LinkedIn or indeed. Where do you find the internship postings

**Reply by tobechinw** [reply]
mostly from this repo tbh

**Comment by ghost** [comment]
until now I didn't recieve nay response from them 😢

**Reply by tobechinw** [reply]
has it been up to two weeks yet? and how’d your own interview go?

**Reply by ghost** [reply]
I didn't recieve any interview yet.

**Reply by tobechinw** [reply]
ohh sorry mann
